### ISSUE TITLE
BUG: Insufficient contrast ratio on Checkout page - Lighthouse 

### ISSUE THREAD

**Issue by evelynfoy** [evelynfoy/the-cheese-and-wine-boutique#70]
The Create an account and login links should be wine so the contrast ratio is sufficient for accessibility

![image](https://user-images.githubusercontent.com/35274182/184549452-f2ccc258-6570-4deb-9dcc-cba218131310.png)

**Comment by evelynfoy** [comment]
Change text-info to text-wine

![image](https://user-images.githubusercontent.com/35274182/185212478-763fcf0c-ac00-4cdb-bdd3-ba829c1464b1.png)

**Comment by evelynfoy** [comment]
![image](https://user-images.githubusercontent.com/35274182/185214895-ad7b6bb4-26da-4898-bb30-71e9a261af1b.png)
